label_studio/io_storages/s3/serializers.py

```diff
@@ -33,6 +34,10 @@ def validate(self, data):
             for key, value in data.items():
                 setattr(storage, key, value)
         else:
+            if 'id' in self.initial_data:
```

Comment:
Was the error occuring here? How do we get into this `else` case? When is `storage` None? Before they have created it, and are trying to validate or what?

Reply:
This code was before, seems that you can provide already created storage to validate connection